wazo_call_logd/tests/test_generator_scenarios.py

```diff
@@ -772,3 +772,224 @@ def test_bad_apple(self, cels: list[CEL]):
+            BRIDGE_EXIT               | 1738431122.459 | 1738431114.455 | 2025-02-01 21:02:10.601229+03:30 | John Conner                | 0012345678910                        | id-1                                 | agentcallback                                             | Local/id-1@agentcallback-00000088;2                                   | Dial            | Local/b464fe7b-bf50-417d-a14b-1bd276f83bf1@usersharedlines,,                                                                                 | {"bridge_id":"dd551bc7-af1b-433b-aa91-bc5cd6b01200","bridge_technology":"simple_bridge"}
+            HANGUP                    | 1738431122.459 | 1738431114.455 | 2025-02-01 21:02:10.601576+03:30 | John Conner                | 0012345678910                        | id-1                                 | agentcallback                                             | Local/id-1@agentcallback-00000088;2                                   |                 |                                                                                                                                              | {"hangupcause":16,"hangupsource":"","dialstatus":"ANSWER"}
+            CHAN_END                  | 1738431122.459 | 1738431114.455 | 2025-02-01 21:02:10.601576+03:30 | John Conner                | 0012345678910                        | id-1                                 | agentcallback                                             | Local/id-1@agentcallback-00000088;2                                   |                 |                                                                                                                                              |
+            BRIDGE_EXIT               | 1738431123.475 | 1738431114.455 | 2025-02-01 21:02:23.441628+03:30 | John Conner                | 0012345678910                        | s                                    | ctx-Company-internal-3344220c-6383-4581-9696-15055ec303e3 | PJSIP/p4go4oxa-00000036                                               | Stasis          | dial_mobile,join,2d3a3a36-93d1-4a6c-8a58-aec01aea8468                                                                                        | {"bridge_id":"wazo-dial-mobile-2d3a3a36-93d1-4a6c-8a58-aec01aea8468","bridge_technology":"simple_bridge"}
+        '''
+    )
+    def test_incoming_call_to_queue(self, cels):
+        call_logs = self.generator.call_logs_from_cel(cels)
+        assert call_logs
+        assert len(call_logs) == 1
+
+        expected_properties = dict(
+            source_name='John Conner',
+            source_exten='0012345678910',
+            requested_exten='8083339999',
+            destination_exten='8083339999',
+            destination_name='SUPPORT',
+            direction='inbound',
+        )
+
+        assert {
+            prop: getattr(call_logs[0], prop) for prop in expected_properties
+        } == expected_properties
```

Comment:
Same as test `test_incoming_call_to_queue_no_answer`, there should be an expectation on `destination_details`.